### PR TITLE
Lint TensorFlow Estimators guide

### PR DESCRIPTION
**Pull request by 8bitmp3**
Splitting PR https://github.com/tensorflow/docs/pull/1736 into smaller digestible bits. Thank you @lamberta @MarkDaoust

### CODE REVIEW

File: site/en/guide/estimator.ipynb [hunk 1]
```diff
@@ -831,20 +819,22 @@
       },
       "source": [
         "<a name=\"estimator_support\"></a>\n",
+        "\n",
         "### What's supported now?\n",
         "\n",
         "There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There may also be a number of bugs in this integration. At this time, we do not plan to actively improve this support, and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to use `tf.distribute` with those APIs instead.\n",
```
Comment (lamberta) [inline site/en/guide/estimator.ipynb:825]:
```suggestion
        "There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There may also be a number of bugs in this integration and there are no plans to improve this support, and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to use `tf.distribute` with those APIs instead.\n",
```

remove the last "we" :)

Comment (8bitmp3) [reply site/en/guide/estimator.ipynb:825]: Good catch, thanks @lamberta. One more thing:

```diff
There is limited support for training with Estimator using all strategies except `TPUStrategy`. Basic training
and evaluation should work, but a number of advanced features such as `v1.train.Scaffold` do not. There
- may also be a number of bugs in this integration and there are no plans to actively improve this support, 
+ may also be a number of bugs in this integration and there are no plans to actively improve this support 
- and instead are focused on Keras and custom training loop support. If at all possible, you should prefer to
+ (the focus is on Keras and custom training loop support). If at all possible, you should prefer to
use `tf.distribute` with those APIs instead.\n",
```

WDYT?

Comment (lamberta) [reply site/en/guide/estimator.ipynb:825]: That works. Thanks!

**Comment by 8bitmp3** [comment]
@lamberta rebased as per your suggestion 👍 CI tests ✅ Thank you, PTAL

**Review by lamberta — APPROVED** [review]
Thanks!

**Pull request merged**